app/kumactl/pkg/output/yaml/printer.go

```diff
@@ -1,14 +1,12 @@
 package yaml
 
 import (
-	"bytes"
 	"io"
 
-	"sigs.k8s.io/yaml"
+	"encoding/json"
+	"gopkg.in/yaml.v2"
```

Comment:
this needs to be explicitly defined in `go.mod` I think

Reply:
Thank you. Added to explicit dependencies in go.mod